src/PowerShellEditorServices.Hosting/Commands/StartEditorServicesCommand.cs

```diff
@@ -38,6 +38,7 @@ public sealed class StartEditorServicesCommand : PSCmdlet
 
         public StartEditorServicesCommand()
         {
+            Environment.SetEnvironmentVariable("POWERSHELL_DISTRIBUTION_CHANNEL", "PSES");
```

Comment:
```suggestion
            //Sets the distribution channel to PSES so starts can be distinguished in PS7+ telemetry
            Environment.SetEnvironmentVariable("POWERSHELL_DISTRIBUTION_CHANNEL", "PSES");
```

Add the comment back